javascripts/map.js

```diff
@@ -1519,8 +1519,11 @@ function printDirections(){
 
 //determine which stylesheet to use, mobile or desktop
 function isMobile() {
-  try{ document.createEvent("TouchEvent"); return true; }
-  catch(e){ return false; }
+	if (navigator.userAgent.match(/Mobi/))
```

Comment:
Follow up question: Did someone specifically ask for a printer button?

How many users wishing to generate a hard copy would find it inconvenient to navigate to the 'Print' dialog on their computer. Assuming they do not CTRL+P or CMP+P